glidertest/plots.py

```diff
@@ -1305,3 +1305,43 @@ def plot_ioosqc(data, suspect_threshold=[25], fail_threshold=[50], title='', ax=
         if force_plot:
             plt.show()
     return fig, ax
+
+def plot_max_depth_per_profile(ds: xr.Dataset, bins= 20, ax = None, **kw: dict) -> tuple({plt.Figure, plt.Axes}):
+    """
+    This function can be used to plot the maximum depth of each profile in a dataset.
+    
+    Parameters
+    ----------
+    ds: xarray on OG1 format containing the profile number and the maximum depth. 
+    bins: int, optional (default=20)
+    
+    Returns
+    -------
+    One figure with two plots illustrating the max depth of each profile and a histogram of the max depths
+
+    Original author
+    ----------------
+    Till Moritz
+    """
+    max_depths = tools.max_depth_per_profile(ds)
+    profile_nums = max_depths.profile_num
+    with plt.style.context(glidertest_style_file):
+        if ax is None:  
+            fig, ax = plt.subplots(1, 2)  
+            force_plot = True
+        else:
+            fig = plt.gcf()
+            force_plot = False
+            
+        ax[0].plot(profile_nums, max_depths,**kw)
```

Comment:
You can avoid creating a new variable and just say:
`ax[0].plot(max_depths.profile_num, max_depths,**kw)`